src/app/blogpost/post-list/post-list.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="post-list">
+  <div class="post-list--container">
```

Comment:
no need to prefix all classes with parent name
you can use nesting in the css / scss
.post-list {
  .container { ... }
  .heading { ... }
  .item { ... }
}

Reply:
yeah, in here thought if I would try arranging styles according to BEM later down the track, but maybe not as it seem to be not stacking well with Angular where every component has its own styles. On the other hand could still be good if I'd make a common styles library and leave component-specific styles for fine-tuning and overrides, still thinking about that while waiting on additional info on what practical part of the training is going to be about